bridges/otelzap/core_test.go

```diff
@@ -31,6 +32,18 @@ func TestCore(t *testing.T) {
 	assert.Equal(t, log.SeverityInfo, got.Severity())
 }
 
+func TestCoreEnabled(t *testing.T) {
+	enabledFunc := func(c context.Context, r log.Record) bool {
+		return r.Severity() >= log.SeverityInfo
+	}
+
+	r := logtest.NewRecorder(logtest.WithEnabledFunc(enabledFunc))
+	zc := NewCore(WithLoggerProvider(r))
+
+	assert.False(t, zc.Enabled(zap.DebugLevel), "level conversion: permissive")
+	assert.True(t, zc.Enabled(zap.InfoLevel), "level conversion: restrictive")
+}
```

Comment:
Can you please test the functionality in a producation-like way (like in `TestCore`) by using https://pkg.go.dev/go.uber.org/zap#Logger.Check?